src/dotnet-libman/Commands/LibmanApp.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.Web.LibraryManager.Tools.Commands
+{
+    internal class LibmanApp : BaseCommand
+    {
+        public LibmanApp(IHostEnvironment hostEnvironment, bool throwOnUnexpectedArg = true) 
+            : base(throwOnUnexpectedArg, "dotnet libman", Resources.LibmanCommandDesc, hostEnvironment)
+        {
+
+        }
+
+        public override BaseCommand Configure(CommandLineApplication parent = null)
+        {
+            base.Configure(parent);
+
+            VersionOption("--version", GetVersion, null);
+
+            Commands.Add(new InitCommand(HostEnvironment).Configure(this));
+            Commands.Add(new InstallCommand(HostEnvironment).Configure(this));
+            Commands.Add(new UninstallCommand(HostEnvironment).Configure(this));
+            Commands.Add(new RestoreCommand(HostEnvironment).Configure(this));
+            Commands.Add(new CleanCommand(HostEnvironment).Configure(this));
+            Commands.Add(new CacheCommand(HostEnvironment).Configure(this));
+
+            // These are shown by child commands.
+            RootDir.ShowInHelpText = false;
+            Verbosity.ShowInHelpText = false;
+
+            return this;
+        }
+
+        private string GetVersion()
+        {
+            FileVersionInfo info = FileVersionInfo.GetVersionInfo(GetType().Assembly.Location);
+            return $"{info.ProductMajorPart}.{info.ProductMinorPart}.{info.ProductBuildPart}.{info.ProductPrivatePart}";
```

Comment:
`info.ProductVersion` will get you the version as a string, including any tags that get generated, e.g. `1.0.20+g3a85c785fc`

Reply:
I was debating whether we should show the git commit id on the help. 
But now I think it might be better to show that (atleast early on). It will make it easier to repro customer issues.